migrate/v0_15/migrate.go

```diff
@@ -174,14 +175,50 @@ func Committee(genesisState v0_14committee.GenesisState) v0_15committee.GenesisS
 			baseStabilityCom := stabilityCom.SetPermissions(newStabilityCommitteePermissions)
 			newStabilityCom := v0_15committee.MemberCommittee{BaseCommittee: baseStabilityCom}
 			committees = append(committees, newStabilityCom)
-		} else {
+		case 2:
 			safetyCom := comtypes.NewMemberCommittee(com.ID, com.Description, com.Members,
 				[]v0_15committee.Permission{v0_15committee.SoftwareUpgradePermission{}},
 				com.VoteThreshold, com.ProposalDuration, v0_15committee.FirstPastThePost)
 			committees = append(committees, safetyCom)
 		}
 	}
 
+	// Initialize hard governance committee
```

Comment:
I would initialize using the 'NewTokenCommittee' method

Reply:
Updated in https://github.com/Kava-Labs/kava/pull/941/commits/25ea9cb6caa3db758836a1f8191ccf586aacae5f